### PR TITLE
Version which works for ELTA daily, pickup and brokenVehicle, SLO-CRO newCLOs, pickup and brokenVehicle

### PR DESCRIPTION
**Pull request by sfabijan**
…t to IntraSofts specs

### CODE REVIEW

File: src/modules/cvrp/processor/vrp_processor.py [hunk 1]
```diff
@@ -387,11 +404,11 @@ def map_slo_cro_vehicles(self, vehicles, event_type):
 
         # First graph is SLO, second graph is CRO
         for v in vehicles:
-            if "S" in v.start_node or "PS" in v.start_node:
+            if "SLO" == v.country:
```
Comment (matejko94) [inline src/modules/cvrp/processor/vrp_processor.py]: maybe add this in constants if you have time..

File: src/modules/cvrp/processor/vrp_processor.py [hunk 2]
```diff
@@ -411,8 +428,8 @@ def map_slo_cro_deliveries(self, deliveries, event_type):
         """
         delivery_parts = [[],[]] # First one is for SLO nodes, Second one is for CRO nodes
         for parcel in deliveries:
-            if "S" in parcel.current_location or "PS" in parcel.current_location:
-                if "H" in parcel.target or "HP" in parcel.target:
+            if "SLO" == parcel.country:
```
Comment (matejko94) [inline src/modules/cvrp/processor/vrp_processor.py]: the same here and on the other places where "CRO" and "SLO" apeared

File: src/modules/utils/structures/parcel.py [hunk 3]
```diff
@@ -4,11 +4,12 @@ class Parcel:
...
     """
-    def __init__(self, uuid, target, volume, current_location, pickup_location=None, destination_location=None, type=None):
+    def __init__(self, uuid, target, volume, current_location, pickup_location=None, destination_location=None, type=None, country=None):
         self.volume = volume
         self.target = target
         self.uuid = uuid
         self.current_location = current_location
         #self.pickup_location = pickup_location
         #self.destination_location = destination_location
-        self.type = type
```
Comment (matejko94) [inline src/modules/utils/structures/parcel.py:14]: remove self.pickup_location and destination_location if they are commented

**Review by matejko94 — APPROVED** [review]

File: src/modules/cvrp/processor/vrp_processor.py (hunk 1 above)
Comment (sfabijan) [reply src/modules/cvrp/processor/vrp_processor.py]: fixed

File: src/modules/utils/structures/parcel.py (hunk 3 above)
Comment (sfabijan) [reply src/modules/utils/structures/parcel.py:14]: fixed

File: src/modules/cvrp/processor/vrp_processor.py (hunk 2 above)
Comment (sfabijan) [reply src/modules/cvrp/processor/vrp_processor.py]: fixed

**Pull request merged**